### DISCUSSION TITLE
How to Distribute TriState Meetings List between 3 Orgs Involved

### DISCUSSION THREAD

**Discussion by ATLCOTech** [code4recovery/12-step-meeting-list#1472] [Q&A]
Hi All, 

I've got a question and I want to make sure my thinking is logical. I currently live in Cincinnati which is part of the Tri State Area of SW Ohio, N. Kentucky and SE Indiana. At some point in time the NKY and SEID intergroup offices closed and Cincinnati Intergroup took up the slack. Now all 3 offices are up and running but all of the meetings are being managed soley by Cincy. They have a shared meeting guide. 

If Cincy sends a filtered out list to NKY and SEID of their meetings (less the Cincy), If Cincy uploaded just their meetings, NKY uploaded theirs and SEID uploaded theirs. Then Cincy reuploaded NKY and SEID (for the purposes of a printed meeting guide) would that give each city the ability to update their own meetings? Currently Cincy is having to do it for all 3. Also, what is the time frame from meetings being removed from the TSML plugin?

This may be better understood with a phone call. My name is Mark 678.368.8368

**Comment by joshreisner** [comment]
sounds like the goal is for AA Cincinnatti to import ID meetings from SEID, and import KY meetings from NKY

they should then do what you say, which is export a CSV and send the ID meetings to SEID, and send the KY meetings to NKY and have them get set up

have those sites then get connected to meeting guide.

once that's done, AA Cincinnatti can delete ID and KY meetings from their site, and wait 12 hours for them to switch to be registered to the other offices

then they can re-import the feeds from other sites using JSON. the meetings will be managed remotely, and they can update by clicking "refresh"

**Comment by ATLCOTech** [comment]
Hey Josh,

Thank you so much for confirming. I'm not familiar with JSON. I've heard of
it after going to the TIAA conference last year. I am a tech enthusiast
though. I don't know how to do any of that part and none of the people I'm
trying to help at the Central Offices do for sure which is how we are where
we are currently.

Is there anyone that can help me walk through this process once it starts?

Sincerely,

Mark J
Tech & Newsletter Chair

If you have a newsletter submission, please send to ***@***.***
.
If you have a website comment, issue or would like to join the tech/web
committee email ***@***.*** ***@***.***>
To contact the steering committee email ***@***.***
Central Office ***@***.***

**Reply by joshreisner** [reply]
sure, we will help you here if you run into problems. i'd just follow the steps above one by one and let us know how it goes
